src/components/skills-quiz/JobCardComponent.jsx

```diff
@@ -32,7 +32,7 @@ const JobCardComponent = ({ jobs, isLoading }) => (
                 {job.job_postings?.length > 0 && (
                   <div className="text-gray-700">
                     <p className="m-0 medium-font">
-                      <span style={{ fontWeight: 700 }}>Median Salary: </span>
+                      <span style={{ fontWeight: 700 }}>Median U.S. Salary: </span>
```

Comment:
We need to make this same change in `SelectJobCard` component as well.

Reply:
done